web/src/components/PoisDesktop.tsx

```diff
@@ -142,6 +144,16 @@ const PoisDesktop = ({
     }
   }, [currentFeatureOnMap, currentPoi, scrollOffset])
 
+  useEffect(() => {
+    if (mapViewRef) {
+      const geolocate = new GeolocateControl({
+        positionOptions: { enableHighAccuracy: true },
+        trackUserLocation: true,
+      })
+      mapViewRef.setGeocontrol('bottom-right', geolocate)
```

Comment:
RTL

Reply:
This is how it was implemented previously ^^. Should we really mirror the map control icons for rtl languages?

Reply:
True, didn't see that, then feel free to ignore. I think we shouldnt mirror the map control icons (no effect anyway since symmetric) but yes, I think we should move them to the left side. Especially bc the zoom buttons already are on the left side